inputs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "pipeline.sampleConfigFiles": ["samples.yml"],
+  "pipeline.samplesConfigs.jar_": "SampleConfig-assembly-0.1.jar",
```

Comment:
I think this should be "pipeline.sampleConfigs.tool_jar" (same goes for the others)

Reply:
:+1: